frontend/src/pages/EditProfilePage/index.jsx

```diff
@@ -61,6 +59,18 @@ const EditProfilePage = ({ uid }) => {
     }
   };
 
+  // This function will set the selectedClass state and call the addClass function as well 
+  const dropDownHandler = (pickClass) => {
+
+    //call the setSelectedClass
+    setClasses(pickClass);
```

Comment:
why do you need to call setClasses here?

Reply:
do we need a state to store the selected class, or simply calling addClass is fine?

Reply:
Ideally, the outside environment only need to know **the selected list** of classes. 
So, within the dropdown component, it could handle all the other local states.
The current selected class is automatically handled by the Select library that you use, so the 
current value will not be needed. 
But it won't have side effects if you choose to manage that state, as long as the DropDown allows the parent component to know what classes are selected.